arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -258,7 +227,10 @@
             "Inherit",
             "Custom"
           ],
-          "type": "string"
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AutomaticTuningMode"
```

Comment:
@tomapaunovic ok, if you want it set to false, it should be fine then. Just to confirm, you would like this property to be modeled as an Enum type in C# for example (for awareness, in this case if you were to add another value later on, you would be making a breaking change).